airflow/api/common/experimental/trigger_dag.py

```diff
@@ -107,7 +107,7 @@ def trigger_dag(
     :param run_id: ID of the dag_run
     :param conf: configuration
     :param execution_date: date of execution
-    :param replace_microseconds: whether microseconds should be zeroed
+    :param replace_microseconds: whether microseconds should be zeroed (ignored if execution_date is given)
```

Comment:
I'm not sure if it matters, but the view in `airflow/www/api/experimental/endpoints.py` doesn't pass through a replace_microseconds parameter, so there's no way of setting this to anything that the default True from the REST API

Reply:
that's true, maybe would be nice to add that as well.